CDP4Composition/Reporting/IReportViewModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+
+namespace CDP4Composition.Reporting
+{
+    public interface IReportViewModel<T> where T : IReportViewModel<T>
```

Comment:
File missing header and docs

Reply:
These files should not even be here any more, as they are leftovers from the initial demo implementation, which are no longer relevant.